allowed.py

```diff
@@ -622,5 +659,5 @@ def read_jupyter_notebook(file_contents: str) -> tuple:
         else:
             print(f"{name}: not a folder, Python file or notebook")
 
-    if reminder:
-        print(reminder)
+    for warning in warnings.values():
+        print(warning)
```

Comment:
This could be `print("WARNING:", warning)` to make strings shorter in the code.

Reply:
Good idea, this is one of those little tricks of the trade to remember!